src/apps/explorer/components/SummaryCardsWidget/index.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { SummaryCards } from './SummaryCards'
+
+import summaryData from './summaryGraphResp.json'
+
+const DELAY_SECONDS = 3 // Emulating API request
+
+export interface BatchInfo {
+  lastBatchDate: Date
+  batchId: string
+}
+
+interface PastAndPresentValue {
+  now: number
+  before: number
+}
+
+interface TotalSummary {
+  batchInfo?: BatchInfo
+  dailyTransactions?: PastAndPresentValue
+  totalTokens?: number
+  dailyFees?: PastAndPresentValue
+}
+
+type RawTotalSummary = Omit<TotalSummary, 'batchInfo'> & {
+  batchInfo: { lastBatchDate: number; batchId: string }
+}
+
+function buildSummary(data: RawTotalSummary): TotalSummary {
+  return {
+    ...data,
+    batchInfo: {
+      ...data.batchInfo,
+      lastBatchDate: new Date(data.batchInfo.lastBatchDate * 1000),
+    },
+  }
+}
```

Comment:
Might consider to have this logic somewhere else, similar to how the API data responses are transformed into internal data objects https://github.com/cowprotocol/explorer/blob/ce1b42f746/src/utils/operator.ts#L299=

Including all the type definitions, of external (the graph, if not provided) and internal stuff

Reply:
I agree!

@ramirotw  can you handle it at #40?